_sass/_partners.scss

```diff
@@ -0,0 +1,91 @@
+@media screen and (min-width: 280px) {
+    .each-partner {
+        width: 100%;
+        background-color: #273647;
```

Comment:
I'd prefer to see these not boxed in, and using the one color versions of the logos. However, if that's going to be challenging because of logo availability, let's just use a white background here `#ffffff`.